doc/index.rst

```diff
@@ -101,18 +118,19 @@ Here are some operations in your client repositories.
 To start using CI config stored in this repo
 --------------------------------------------------
 
-With the following few short steps, you can start in your client repository using CI confiurations stored in here (`industrial_ci` repository).
+With the following few short steps, you can start in your client repository using CI configurations stored in here (`industrial_ci` repository).
 
-1. Don't forget to activate Travis CI on your github repository (you may do so on https://travis-ci.org/profile/YOUR_GITHUB_ORGANIZATION or https://travis-ci.org/profile/YOUR_GITHUB_USER).
+1. Don't forget to activate CI for your repository.
 
-2. In `.travis.yml` file in your client repo, add in `before_config` section a sentence `git clone https://github.com/ros-industrial/industrial_ci.git .ci_config`, like below:
+- For Travis CI and GitHub, you may do so on https://travis-ci.org/profile/YOUR_GITHUB_ORGANIZATION or https://travis-ci.org/profile/YOUR_GITHUB_USER (replace capital with your value).
 
+2. In `CI config <#terminology>`_ file in your client repo, add a sentence `git clone https://github.com/ros-industrial/industrial_ci.git .ci_config`, like below:
 ::
 
-  before_config:
+  install:
     - git clone https://github.com/ros-industrial/industrial_ci.git .ci_config
```

Comment:
Note: We should change this to `.industrial_ci` in the future. (new template in #105)

Reply:
> Note: We should change this to .industrial_ci in the future. (new template in #105)


Why is that?

Reply:
Because it's no configuration at all ;)